test/e2e/network/netpol/kubemanager.go

```diff
@@ -254,9 +265,14 @@ func (k *kubeManager) waitForHTTPServers(model *Model) error {
 	const maxTries = 10
 	framework.Logf("waiting for HTTP servers (ports 80 and 81) to become ready")
 
+	protocols := model.Protocols
+	if framework.NodeOSDistroIs("windows") {
```

Comment:
So windows only supports TCP?  (or the only protocol we're going to test on windows is TCP?)

This decision should be made where the model is instantiated, otherwise the model is out of sync with what's actually happening = confusion.